adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -71,11 +72,15 @@ - (void)renderBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguratio
     _loadCompletionHandler(nil, error);
     return;
   }
-
-  _nativeExpressBannerView =
-      [[BUNativeExpressBannerView alloc] initWithSlotID:placementId
-                                     rootViewController:adConfiguration.topViewController
-                                                 adSize:_bannerSize];
+    
+  GADPangleNetworkExtras *extras = adConfiguration.extras;
+    
+  BUAdSlot *slot = [BUAdSlot new];
+  slot.ID = placementId;
+  slot.userData = extras.userDataString;
```

Comment:
If the `extras` is null and we try to access the `userDataString` property, I would assume it's a no-op.

Reply:
ok~